### PR TITLE
trivial: Show the device branch in fwupdmgr

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-util-common.c [hunk 1]
```diff
@@ -1217,6 +1217,13 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 		fu_common_string_append_kv (str, idt + 1, _("Vendor"), tmp2);
 	}
 
+	/* branch */
+	if (fwupd_device_get_branch (dev) != NULL) {
+		/* TRANSLATORS: the stream of firmware, e.g. nonfree or open-source */
+		fu_common_string_append_kv (str, idt + 1, _("Branch"),
```
Comment (superm1) [inline src/fu-util-common.c:1223]: As this is an attribute of the firmware installed not the device I wonder if you want to prefix this with "Firmware", IE `Firmware Branch`

Comment (hughsie) [reply src/fu-util-common.c:1223]: Or `Release Branch` perhaps?

Comment (superm1) [reply src/fu-util-common.c:1223]: Yeah that's good

Comment (hughsie) [reply src/fu-util-common.c]: Fixed.

**Review by superm1 — APPROVED** [review]

**Pull request merged**